include/dlr.h

```diff
@@ -324,8 +368,17 @@ int GetDLROutputSizeDim(DLRModelHandle* handle, int index, int64_t* size,
  message.
  */
 DLR_DLL
-int GetDLROutputType(DLRModelHandle* handle, int index,
-                     const char** output_type);
+int GetDLROutputType(DLRModelHandle* handle, int index, const char** output_type);
+
+/*!
+ * \brief Gets the index-th output from the model.
+ * \param handle The model handle returned from CreateDLRModel().
+ * \param index The index-th output.
+ * \param out The pointer to save the output DLTensor
+ * \return 0 for success, -1 for error. Call DLRGetLastError() to get the error message.
+ */
+DLR_DLL
+int GetTVMOutputTensor(DLRModelHandle* handle, int index, void* dltensor);
```

Comment:
Mention how this can only be used for TVM graph runtime models (such as those created by CreateTVMModel)

Reply:
done